### DISCUSSION TITLE
ES6 Hmr with webpack on Node

### DISCUSSION THREAD

**Discussion by Olli1080** [webpack/webpack#15587] [Q&A]
I've been trying to upgrade my ssr project to use es6 on both the server and client but ran into an issue with hmr. On changes the compilation fails at this line https://github.com/webpack/webpack/blob/86a8bd9618c4677e94612ff7cbdf69affeba1268/lib/esm/ModuleChunkFormatPlugin.js#L52 due to wip. Can i somewhere see how much progress has been made on this so far?

Thank you for your response in advance.

**Comment by Isaac-Leonard** [comment]
I've also come across this issue as of today, it seems as though it is still not implemented

**Comment by 42shadow42** [comment]
Found this project representing the known pieces of the feature, not sure it's actively being worked though....

https://github.com/webpack/webpack/projects/8
